### PR TITLE
GIX-1097: Update candid canisters

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Upgrade candid files to commit that will be released Tuesday 1st of November.

# Changes

* Update candid files and related.

# Tests

* Change sns-wasm mock to adapt to new response type.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

I updated the candid files to tomorrow's release commit: https://dfinity.slack.com/archives/C02P68W6MQA/p1666986157237939

File: packages/sns/candid/sns_index.did [hunk 1]
```diff
@@ -47,7 +47,7 @@ type TransactionWithId = record {
 type GetTransactions = record {
   transactions : vec TransactionWithId;
   // The txid of the oldest transaction the account has
-  oldest_tx_id : opt nat64;
+  oldest_tx_id : opt TxId;
```
Comment (peterpeterparker) [inline packages/sns/candid/sns_index.did:50]: In your PR #219 you had issue with `TxId` which if I reckon you had to manually overwrite. Is it safe to have here `TxId` again?

Comment (lmuntaner) [reply packages/sns/candid/sns_index.did:50]: Yes when using the new SNS Index canisters. I tested it against small11 and it was working. I saw a transactions and everything worked well.

The problem is that to work with transactions we can only work with small11 at the moment. Which is the only environment with the latest sns canisters.

**Review by peterpeterparker — APPROVED** [review]
LGTM

File: packages/sns/candid/sns_index.did (hunk 1 above)
Comment (peterpeterparker) [reply packages/sns/candid/sns_index.did:50]: Ok

Comment (peterpeterparker) [reply packages/sns/candid/sns_index.did:50]: Thanks for the feedback

**Pull request merged**